customization/halium/package-lists/halium.list.chroot

```diff
@@ -1 +1,5 @@
-libhybris-git
```

Comment:
NetworkManager should probably be also included in the rootfs (to match Halium docs)

Reply:
strace should also be added

Reply:
libhybris-aarch64-git is missing a wayland dependency